### PR TITLE
Update | Move Data CSVs to Own Repo

### PR DESCRIPTION
**Pull request by davidebukali**
#424 Move csv files from the data_updates folder to a separate [git repo](https://github.com/devinit/ddw-data-update-configs)

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -56,15 +56,25 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
+    - name: Create and start virtual environment
+      run: |
+        python3 -m venv venv
+        source venv/bin/activate
```
Comment (edwinmp) [inline .github/workflows/CI.yml]: What's the point of this when everything is being run in docker?

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -56,15 +56,25 @@ jobs:
...
+
     - name: Install dependencies
       run: |
         pip3 install --user codecov
+        pip3 install -r requirements.txt
```
Comment (edwinmp) [inline .github/workflows/CI.yml]: This too? Sure it happens during the docker build in the next step.

File: README.md [hunk 3]
```diff
@@ -19,6 +19,17 @@ The new and improved DDW Analyst UI interface
...
 
+### Import CSV Files
+
+1. Manually run management command to download csv files:
+
+        docker-compose exec web python manage.py update_csv_files
+
+2. Create a scheduled event to periodically download updates from the git repo. The bash script is `update_csv_files.sh`
+
+3. The git hub repo with csv files can be found here https://github.com/devinit/ddw-data-update-configs
```
Comment (edwinmp) [inline README.md]: Put the link in the `here` string ... i.e [here](https://github.com/devinit/ddw-data-update-configs)

File: requirements.txt [hunk 4]
```diff
@@ -23,3 +23,4 @@ rest-social-auth>=4.2.0
 six==1.12.0
 soupsieve==1.9
 SQLAlchemy==1.3.2
+PyGithub==1.54.1
```
Comment (edwinmp) [inline requirements.txt]: If you look carefully, you'll see that our dependencies are alphabetised :)

**Review by edwinmp — CHANGES_REQUESTED** [review]

File: .github/workflows/CI.yml (hunk 2 above)
Comment (davidebukali) [reply .github/workflows/CI.yml]: Also removed, the docker installation was sufficient

File: .github/workflows/CI.yml (hunk 1 above)
Comment (davidebukali) [reply .github/workflows/CI.yml]: Removed this step

File: README.md (hunk 3 above)
Comment (davidebukali) [reply README.md]: Updated

File: requirements.txt (hunk 4 above)
Comment (davidebukali) [reply requirements.txt]: Updated

**Review by edwinmp — APPROVED** [review]
Would rather merge them in separately, so approved!

**Pull request merged**